app/api-v1/controllers/Recipe/index.js

```diff
@@ -0,0 +1,51 @@
+const { runProcess } = require('../../../utils/dscp-api')
+const { client } = require('../../../db')
+const { mapRecipeData } = require('./helpers')
+const { BadRequestError, NotFoundError } = require('../../../utils/errors')
+
+module.exports = {
+  transaction: {
+    create: async (req) => {
+      const service = '/recipe/{id}/creation'
+      const { id } = req.params
+      if (!id)
+        throw new BadRequestError({
+          message: 'missing [id] param from request',
+          service,
+        })
+
+      const [recipe] = await client.from('recipes').select('*').where({ id })
+      if (!recipe)
+        throw new NotFoundError({
+          message: 'recipe not found',
+          service,
+        })
+
+      const payload = {
+        inputs: [],
+        outputs: [
+          {
+            roles: { Owner: recipe.supplier },
```

Comment:
this looks wrong. The `self` address should be the `Customer` and the `supplier` should be the `AdditiveManufacturer`

Reply:
Might need to get a bit more clarity on this one. Where do we pull AddiitiveManufacturer, is this is where transactions review comes in?